### ISSUE TITLE
Error: Unknown enum value. (Parameter 'trip') Actual value was eigth.

### ISSUE THREAD

**Issue by AdamSEY** [rageagainsthepc/GuitarPro-to-Midi#22]
[Difficult_To_Cure___Live_At_Budokan__Tokyo__Japan__1984 (2).gp.zip](https://github.com/rageagainsthepc/GuitarPro-to-Midi/files/10351037/Difficult_To_Cure___Live_At_Budokan__Tokyo__Japan__1984.2.gp.zip)

**Comment by rageagainsthepc** [comment]
I'm missing a little bit of context here. Does the program abort when it fails to convert `trip` to an enum?

**Comment by AdamSEY** [comment]
That's completely right! You can try it.

**Comment by rageagainsthepc** [comment]
I agree the program should continue instead of simply exiting with an error, but usually when we encounter these kinds of errors it's not caused by invalid data. I think it's most likely something that simply isn't implemented yet. We should at least issue a warning here instead of silently skipping over this measure. Feel free to create a PR and I will review it. If you know what the meaning of this unknown value is you could also try to implement that.

**Comment by AdamSEY** [comment]
Please take a look: https://github.com/rageagainsthepc/GuitarPro-to-Midi/pull/23

**Comment by rageagainsthepc** [comment]
That PR was basically finished except for some minor formatting issues when the author simply stopped responding.

**Comment by AdamSEY** [comment]
@rageagainsthepc, I'm sorry I really got busy and forgot about this, I will take a look.
